Source/2_Core/Models/LeaderboardsInfoResponse.cs

```diff
@@ -63,6 +67,17 @@ internal struct QualificationInfo {
         public string approvers;
     }
 
+    public struct ClanRankingInfo
+    {
+        public ClanRankingClanInfo clan;

```

Comment:
instead of `ClanRankingClanInfo`, you can use `Clan` it is already exists in models

Reply:
Fixed; I think I intended for some clanRanking data that doesn't exist in "Clan" to be useable by the mod, but scrapped it for this PR.